components/applications-service/integration_test/storage_service_timestamp_test.go

```diff
@@ -0,0 +1,113 @@
+//
+//  Author:: Salim Afiune <afiune@chef.io>
+//  Copyright:: Copyright 2019, Chef Software Inc.
+//
+
+package integration_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStorageGetServiceFromUniqueFieldsNotExist(t *testing.T) {
+	svc, exist := suite.StorageClient.GetServiceFromUniqueFields("foo", "123")
+	assert.False(t, exist)
+	assert.Nil(t, svc)
+}
+
+func TestStorageGetServiceFromUniqueFieldsEmptyParameters(t *testing.T) {
```

Comment:
This test doesn't seem to be testing much since the storage is empty. Maybe you could insert the hab event from the next test and then check that the following all return false:

```
suite.StorageClient.GetServiceFromUniqueFields("", "")
suite.StorageClient.GetServiceFromUniqueFields("postgres", "")
suite.StorageClient.GetServiceFromUniqueFields("", "1q2w3e4r")
```

Reply:
Will do

Reply:
Though, now that I am doing this modification, it doesn't matter if we have something in the database or not since we have verifications that check if any of the parameters is empty, we just return immediately without running a query. Should I still modify it?